plugin/pom.xml

```diff
@@ -46,6 +46,22 @@
   </developers>
 
   <dependencies>
+    <dependency>
+      <groupId>org.everit.json</groupId>
+      <artifactId>org.everit.json.schema</artifactId>
+      <version>1.3.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.5</version>
+    </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
```

Comment:
Should it actually be a part of https://github.com/jenkinsci/jackson2-api-plugin or BOM? There are several dependences including production code CC @jvz @jglick